modules/post/multi/gather/tomcat_gather.rb

```diff
@@ -62,14 +60,24 @@ def report_creds(user, pass, port)
 
   def gatherwin()
     print_status("Windows OS detected, enumerating services")
+    tomcatHomeArray = Array.new
     service_list.each do |service|
       if service[:name].downcase().include? "tomcat"
         print_good("Tomcat service found")
-        tomcat_home = service_info(service[:name])[:path].split("\\bin\\")[0]
-        conf_path = tomcat_home.split('"')[1] + "\\conf\\tomcat-users.xml"
+        tomcatHomeArray.push(service_info(service[:name])[:path].split("\\bin\\")[0])
+      end
+    end
+      
+    if tomcatHomeArray.size > 0
+      tomcatHomeArray.each do |tomcat_home|
+        if tomcat_home.include? '"'
+          tomcat_home = tomcat_home.split('"')[1]
+        end
+
+        conf_path = tomcat_home + "\\conf\\tomcat-users.xml"
 
         if exist?(conf_path)
-          print_status("tomcat-users.xml found")
+          print_status(conf_path + "found!")
```

Comment:
I would change this to `print_status("#{conf_path} found!")` that way you use templating and adds a space between the file and the word found.